src/nimblepkg/options.nim

```diff
@@ -67,16 +67,16 @@ type
     of actionCustom:
       command*: string
       arguments*: seq[string]
-      flags*: StringTableRef
+      flags*: seq[string]
```

Comment:
Why the change to seq? A table is more flexible and supports ``--key:val``.

Reply:
Reusing the work done in `nimble run` as well as the `getFlagString()` code - there was no reason to store some flags as seqs and this one as a table.

Reply:
Shouldn't they all be stored as tables to enable `--key:val`?

Reply:
`--key:val` is supported by this method since it is used by `nimble run` as well.